printing/proof_diffs.ml

```diff
@@ -263,14 +263,12 @@ let pr_econstr_env ?lax ?inctx ?scope env sigma t =
 let pr_lconstr_env ?lax ?inctx ?scope env sigma c =
   pr_leconstr_env ?lax ?inctx ?scope env sigma (EConstr.of_constr c)
 
-let diff_concl ?og_s nsigma ng =
+let diff_concl ?og_s ng =
   let o_concl_pp = match og_s with
-    | Some (og, osigma) ->
-      let { ty = oty; env = oenv } = process_goal osigma og in
-      pp_of_type oenv osigma oty
-    | None -> Pp.mt()
+  | Some { ty = oty; env = oenv; sigma = osigma } -> pp_of_type oenv osigma oty
+  | None -> Pp.mt()
   in
-  let { ty = nty; env = nenv } = process_goal nsigma ng in
+  let { ty = nty; env = nenv; sigma = nsigma } = ng in
```

Comment:
Quite inconsistent that the variable being assigned (e.g. nty) is on the **right-hand** side in this construct.  Another OCaml quirk.

Reply:
Is it that quirky? Sure its a bit abusive to use `=` inside records, but its no different than having written`let (nty,nenv,nsigma) = ...`?